package.json

```diff
@@ -1,8 +1,91 @@
 {
   "name": "daskeyboard-applet--flight-price-watcher",
   "version": "1.0.0",
+  "displayName": "Flight Price Watcher",
+  "description": "Find the best prices and throw yourself in the air with Flight Price Watcher.",
+  "longDescription": "Displays a notification on Das Keyboard Q series when there is a cheap flight.",
+  "publisher": "Das Keyboard",
+  "officialProductName": "Flight Price Watcher",
+  "publisherName": "Das Keyboard",
+  "isSingleton": false,
+  "icon": "assets/icon.png",
+  "image": "assets/image.png",
+  "authorName": "Das Keyboard",
+  "authorUrl": "https://twitter.com/daskeyboard",
+  "issuesUrl": "https://github.com/daskeyboard/daskeyboard-applet--flight-price-watcher/issues",
+  "homePageUrl": "https://github.com/daskeyboard/daskeyboard-applet--flight-price-watcher",
+  "developerRepoUrl": "https://github.com/daskeyboard/daskeyboard-applet--flight-price-watcher",
+  "licenseUrl": "http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt",
+  "changelogUrl": "CHANGELOG.md",
+  "readMeUrl": "README.md",
+  "readMeEndUserUrl": "README_ENDUSER.md",
+  "engines": {
+    "das-keyboard-q": "3.2.6"
+  },
   "main": "index.js",
-  "repository": "git@github.com:daskeyboard/daskeyboard-applet--flight-price-watcher.git",
-  "author": "Das Keyboard",
-  "license": "MIT"
+  "scripts": {
+    "test": "mocha"
+  },
+  "author": "Camille Bouquet",
+  "license": "MIT",
+  "dependencies": {
+    "daskeyboard-applet": "^2.11.4",
+    "localStorage": "^1.0.4"
+  },
+  "qConfig": {
+    "geometry": {
+      "width": 1,
+      "height": 1,
+      "origin": {
+        "x": 1,
+        "y": 1
+      }
+    },
+    "authorization": {
+      "type": "apiKey",
+      "hint": "Get the api key from the search flight page of Skyscanner website.",
+      "supportUrl": "https://skyscanner-skyscanner-flight-search-v1.p.rapidapi.com/apiservices/pricing/v1.0"
+    },
+    "questions": [
+      {
+        "key": "ORIGIN_PLACE",
+        "label": "Choose an origin place.",
+        "help": "",
+        "required": true,
+        "order": 1,
+        "value": "SFO",
+        "controlType": "textbox"
```

Comment:
Would be cool if you can use dynamic search. But we will see that later @Mubramaj Memo